cmd/frontend/graphqlbackend/repositories_test.go

```diff
@@ -494,6 +680,19 @@ func TestRepositories_Integration(t *testing.T) {
 			t.Fatal(err)
 		}
 
+		if rsc.indexed {
+			repo, err := db.Repos().GetByName(ctx, rsc.repo.Name)
+			if err != nil {
+				t.Fatal(err)
+			}
+			err = db.ZoektRepos().UpdateIndexStatuses(ctx, map[uint32]*zoekt.MinimalRepoListEntry{
+				uint32(repo.ID): {},
```

Comment:
After the `create` the `ID` on the `rsc.repo` is set, so no need to query it again:

```suggestion
			err := db.ZoektRepos().UpdateIndexStatuses(ctx, map[uint32]*zoekt.MinimalRepoListEntry{
				uint32(rsc.repo.ID): {},
```